packages/cli/lib/projects.js

```diff
@@ -330,6 +338,31 @@ const makeGetTaskStatus = taskType => {
               spinnies.fail('overallTaskStatus', {
                 text: statusStrings.FAIL(taskName),
               });
+
+              const failedSubtask = subTaskStatus.filter(
+                subtask => subtask.status === 'FAILURE'
+              );
+
+              logger.log('-'.repeat(50));
+              logger.log(
+                `${statusStrings.SUBTASK_FAIL(
+                  buildId || taskId,
+                  failedSubtask.length === 1
+                    ? failedSubtask[0][statusText.SUBTASK_NAME_KEY]
+                    : failedSubtask.length + ' components'
+                )}\n`
+              );
+              logger.log('See below for a summary of errors.');
+              logger.log('-'.repeat(50));
```

Comment:
Not necessary to do in this PR, but seeing your recent PR that added a "ui.js" file makes me think that things like this horizontal line could maybe go in there.

Reply:
100%